cdap-cli/src/main/java/co/cask/cdap/cli/CLIMain.java

```diff
@@ -77,34 +77,34 @@
   private static final String TOOL_NAME = "cli";
 
   @VisibleForTesting
-  public static final Option HELP_OPTION = new Option(
+  static final Option HELP_OPTION = new Option(
     "h", "help", false, "Print the usage message.");
 
-  @VisibleForTesting
-  public static final Option URI_OPTION = new Option(
+  private static final Option URI_OPTION = new Option(
     "u", "uri", true, "CDAP instance URI to interact with in" +
     " the format \"[http[s]://]<hostname>[:<port>[/<namespace>]]\"." +
     " Defaults to \"" + getDefaultURI().toString() + "\".");
 
-  @VisibleForTesting
-  public static final Option VERIFY_SSL_OPTION = new Option(
+  private static final Option VERIFY_SSL_OPTION = new Option(
     "v", "verify-ssl", true, "If \"true\", verify SSL certificate when making requests." +
     " Defaults to \"" + DEFAULT_VERIFY_SSL + "\".");
 
   @VisibleForTesting
-  public static final Option AUTOCONNECT_OPTION = new Option(
+  static final Option AUTOCONNECT_OPTION = new Option(
     "a", "autoconnect", true, "If \"true\", try provided connection" +
     " (from " + URI_OPTION.getLongOpt() + ")" +
     " upon launch or try default connection if none provided." +
     " Defaults to \"" + DEFAULT_AUTOCONNECT + "\".");
 
-  @VisibleForTesting
-  public static final Option DEBUG_OPTION = new Option(
+  private static final Option DEBUG_OPTION = new Option(
     "d", "debug", false, "Print exception stack traces.");
 
   private static final Option SCRIPT_OPTION = new Option(
     "s", "script", true, "Execute a file containing a series of CLI commands, line-by-line.");
 
+  private static final Option TIMEOUT_OPTION = new Option(
+    "t", "timeout", true, "Sets read timeout of the CLI in seconds.");
```

Comment:
I think it'd be more informative if it was "http read timeout" instead of just "read timeout"